apps/explorer/src/pages/epochs/EpochDetail.tsx

```diff
@@ -47,8 +49,12 @@ export default function EpochDetail() {
         })
     );
 
+    const { data: currentEpoch } = useQuery(['epochs', 'current'], async () =>
+        enhancedRpc.getCurrentEpoch()
+    );
+
     const [epochData] = data?.data ?? [];
-    const isCurrentEpoch = !epochData?.endOfEpochInfo;
```

Comment:
some epochs don't have `endOfEpochInfo` so we were displaying them as `In Progress`. might make sense to live with that if we want to avoid the additional rpc call